functions/scraper/func.py

```diff
@@ -0,0 +1,60 @@
+import ujson
+import os
+import flickrapi
+import random
+import requests
+import fdk
+
+from cloudevents.sdk import converters
+from cloudevents.sdk import marshaller
```

Comment:
Imported but not used

Reply:
well I presume I need to use those if I want to send the event the right way (not as json)